x-pack/plugins/synthetics/e2e/journeys/synthetics/alerting_default.journey.ts

```diff
@@ -42,7 +42,7 @@ journey('AlertingDefaults', async ({ page, params }) => {
     await page.click('.euiComboBox__inputWrap');
     await page.click("text=There aren't any options available");
     await page.click('button:has-text("Add connector")');
-    await page.click('p:has-text("Slack")');
+    await page.click('p:has-text("Slack Webhook")');
```

Comment:
hmmm ... I have no idea what this UX is actually doing, but I'm wondering if how this is going to work.  Does it display a list of all the connector types, so would show both Slack's?  Or will we need to do the "group" thing here also?  Or maybe it's all just working because they end up going through our common framework code.